src/citrine/resources/gemd_resource.py

```diff
@@ -22,17 +23,39 @@
 class GEMDResourceCollection(DataConceptsCollection[DataConcepts]):
```

Comment:
@anoto-moniz here we will need to discuss how to pick the appropriate API endpoint for building the table depending on the desired execution scope (whole team or project), I've sent you a pm to start talking about it.